docs/content/v2/en/options.md

```diff
@@ -189,6 +189,31 @@ List of all available font files of a font family variation.
 | `src`  | `String` | yes      | path to a font file, the use of aliases is possible        |
 | `type` | `String` | yes      | file format of the specified file, e.g. `woff`, `woff2`, … |
 
+## `pictureFormats`
+- Type: `Array`
+  - Default: `['webp', 'avif', 'jpg|jpeg|png|gif']`
+
+Defines the default `formats` for the `SpeedkitPicture`.
+
+Can be overwritten in the `SpeedkitPicture`.
+
+By default, the operator `|` is used to combine formats.
+
+For each `png` a `jpg` and for each `jpg` a `png` is generated.
```

Comment:
"The declaration below generates a PNG, JPEG and GIF (destination format) for each JPEG (source format). The same applies to a PNG and a GIF as source format. However, this is not practical for the source specifications in the Picture."